public/app/frontend/style.scss

```diff
@@ -310,3 +311,43 @@ $iconSizes: (
         }
     }
 }
+
+// From GDS https://design-system.service.gov.uk/styles/layout/
+@mixin visually-hidden($important: true) {
+    position: absolute if($important, !important, null);
+
+    // Absolute positioning has the unintended consequence of removing any
+    // whitespace surrounding visually hidden text from the accessibility tree.
+    // Insert a space character before and after visually hidden text to separate
+    // it from any visible text surrounding it.
+    &::before {
+        content: "\00a0";
+    }
+
+    &::after {
+        content: "\00a0";
+    }
+
+    width: 1px if($important, !important, null);
+    height: 1px if($important, !important, null);
+    // If margin is set to a negative value it can cause text to be announced in
+    // the wrong order in VoiceOver for OSX
+    margin: 0 if($important, !important, null);
+    padding: 0 if($important, !important, null);
+
+    overflow: hidden if($important, !important, null);
+    clip: rect(0 0 0 0) if($important, !important, null);
+    clip-path: inset(50%) if($important, !important, null);
+
+    border: 0 if($important, !important, null);
+
+    // For long content, line feeds are not interpreted as spaces and small width
+    // causes content to wrap 1 word per line:
+    // https://medium.com/@jessebeach/beware-smushed-off-screen-accessible-text-5952a4c2cbfe
+    white-space: nowrap if($important, !important, null);
+
+    // Prevent users from selecting or copying visually-hidden text. This prevents
+    // a user unintentionally copying more text than they intended and needing to
+    // manually trim it down again.
+    user-select: none;
```

Comment:
As some of this is copied from GDS, would it be a bad idea to install the GDS (or even MoJ) design system npm package and import from it?

Reply:
@EarthlingDavey I've tried to keep it separate as much as possible because the justice site has it's own visual identity and it's only a couple of things that we're using from the design system. 

I'm hoping in future we can use the GDS or MoJ design system but given the concerns around any major visual changes I think it's best to try to keep them apart in these initial stages, though I'll revisit it if we find we're using a lot from them.

Reply:
That sounds good. Would be nice to see the site head in the direction of GDS or MoJ _at some point_.